docs/Development-Code_Review_Procedures_and_Guidelines.md

```diff
@@ -46,9 +46,9 @@
author already considered them. ("What do you think about using a custom validator here?")</li><li>If discussions turn too theoretical or touch big architectural questions, move the discussion offline. In the meantime, let the author make the final decision on alternative implementations.</li><li>Do not keep the code hostage. Keep in mind the context and the most important thing - does it work?</li><h2 id="Merging-a-merge-request"><strong>Merging a merge request</strong></h2><h4 id="Before-making-the-decision-to-merge">Before making the decision to merge:</h4><li>Set a milestone.</li><li>Consider warnings and errors from Github CI bots, code quality bots, and other reports. Unless a strong case can be made for the violation, these should be resolved before merging. A comment must be posted if the PR is merged with any failed job.</li><li>If the PR contains both quality and non-quality-related changes, the PR should be merged by the relevant maintainer or senior software engineer after the quality related changes are approved by more then one software engineer.</li><br><p>If a pull request is fundamentally ready, but needs only trivial fixes (such as typos), consider demonstrating a bias for action by making those changes directly without going back to the author. You can do this by using the suggest changes feature to apply your own suggestions to the pull request.</p><p><b>Note that:</b></p><li>merging is limited to Armbian Github Organization members, you can  <a href="https://www.armbian.com/contact/" target="_blank" rel="noopener">apply here</a> to become one, if you have not already been invited.</li><li>If the changes are not straightforward, please prefer allowing the author to make the change.</li><li>Before applying suggestions, edit the pull request to make sure squash and merge is enabled</li><br><p><em><strong>Authors are not authorized to merge their own pull requests and need to seek approval from another maintainer / developer to merge.</strong></em></p><h2 id="Armbians-Assistance"><strong>Armbian's Assistance</strong></h2><p></p><p>If you have questions about being a developer or want to learn more and deeper insights about the build framework, Armbian will try to guide you to the appropriate documentation or information in a <em>best-effort</em> fashion. If time allows, at our descrection, we will try our best to explain and teach you personally various aspects about our processes. If best effort guidance is not enough, contact us for <a href="https://www.armbian.com/contact/" target="_blank" rel="noopener">professional assistance</a>.</p><p>If you have any concerns please do not hesitate to reach out via forums, IRC or Discord. <em><strong>Armbian cares about the people who care about Armbian</strong></em>&nbsp; <span><img alt=":)" data-emoticon="true" height="20" src="https://forum.armbian.com/uploads/emoticons/default_smile.png" title=":)" width="20"></span></p><h3 id="References--Sources"><i>References / Sources:</i></h3><ul>
```

Comment:
Can you sum up what exactly you adjusted since how Github shows it is rather useless

Reply:
Two times added a slash `\` after `https://www.armbian.com/contact`